features/0755-oca-for-aries/README.md

```diff
@@ -0,0 +1,402 @@
+with "multilingual" can have multiple instances of the overlay, one for each issuer-supported
+language (English, French, Spanish, Braile, etc.).
+
+- the [character_encoding
+overlay](https://oca.colossi.network/v1.1.0-rc.html#character-encoding-overlay)
+contains the encoding (usually "utf-8" or "base64") for each attribute in the
+capture base.
+- the [format overlay](https://oca.colossi.network/v1.1.0-rc.html#format-overlay) defines an input structure
+for the data. The format may be useful in displaying the data in a style expected by the user, based on
+the content of the data and their language preferences. For example, displaying the elements of a date
+in the user's preferred format.
+- the multilingual [label overlay](https://oca.colossi.network/v1.1.0-rc.html#label-overlay) defines a
+label to be used for each attribute for a given language. The label overlay also includes labels for
+attributes with enumerated values. For example, if a data attribute might contains the codes "EN", "FR", "SP"
+that mean "English", "French" and "Spanish", respectively.
+- the multilingual [information
+overlay](https://oca.colossi.network/v1.1.0-rc.html#information-overlay) defines
+a description or help text about each attribute for a given language. There will
+be one overlay per issuer-supported language.
+- the multilingual [meta overlay](https://oca.colossi.network/v1.1.0-rc.html#meta-overlay) contains
+information about the credential itself, such as "name" and "description."
+
+##### The AnonCreds "dateint" Format
+
+A common data format used in Aries AnonCreds verifiable credentials is "`dateint`", as
+described in [RFC 0592](../0592-indy-attachments/README.md). A `dateint` is a date stored
+as an integer (for example `September 29, 2022` is the integer `20,220,929`). In the format
+overlay, `dateint` is a permitted value when used with Aries. An recipient of an OCA Bundle
+with a `dateint` SHOULD extract from the integer data value the year, month and day and
+display those elements in the style appropriate for the user. For example, a mobile app
+should display the data as a date based on the user's language/country setting.
+
+#### Aries Specific Overlays
+
+In addition to the core OCA Overlays listed above, Aries agents also use an
+additional "extension" overlay, the [credential_branding overlay](#), that
+provides the issuer with a way to provide a series of data elements about the
+credential branding that they would like to see applied. The credential_branding
+overlay provides a list of name/value pairs, and the holders (and verifiers) can
+use the values when displaying the credential of that type from that issuer. The
+holder/verifier controls the details of how the credentials are displayed, but
+the issuer does have some input into the branding. The following are the branding
+data elements the issuer can set:
+
+- logo_uri - a URI pointing to a logo to display with the credential.
```

Comment:
Can we assume that the `logo` and `background_image` is an URL? embedded base64 provided as [Data URLs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs) (e.g.: `data:image/png;base64,...`)